app/controllers/root_controller.rb

```diff
@@ -0,0 +1,8 @@
+class RootController < ApplicationController
+  def index
+    respond_to do |format|
+      format.html # Render the HTML template
+      format.json { render json: { message: 'JSON response' } }
+    end
+  end
+end
```

Comment:
- [ ] Great work on having an index method but this index method should be empty for now because it is rendering the react application so it would be nice if you remove all the code inside the index method :+1:

Reply:
thanks resolved